<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>passwordchange</class>
 <widget class="QMainWindow" name="passwordchange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Password Change</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>New Pass</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Confirm</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="new_password"/>
       </item>
       <item>
        <widget class="QLineEdit" name="confrim_password"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="password_confirm_button">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QLabel" name="passchange_error">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
